pkg/controller/certificates/sync.go

```diff
@@ -407,13 +411,25 @@ func (c *certificateRequestManager) processCertificate(ctx context.Context, crt
 	}
 
 	reason := apiutil.CertificateRequestReadyReason(existingReq)
+
+	// If the CertificateRequest condition is present and has the status of
+	// "True" then do not attempt to retry the CertificateRequest. Else we can
+	// retry.
+	if apiutil.CertificateRequestHasInvalidRequest(existingReq) {
+		log.Info("CertificateRequest is in an InvalidRequest state and will no longer be processed", "state", reason)
```

Comment:
Can you change the log key `state` here to be `reason` and add a `message` key with the contents of the message?